.changeset/brave-days-happen.md

```diff
@@ -0,0 +1,6 @@
+---
+"docs": minor
+"sit-onyx": minor
```

Comment:
```suggestion
```

There are no relevant changes for the users, right? So we don't need to publish a new onyx version